src/lib.rs

```diff
@@ -33,6 +33,7 @@ pub use crate::shims::foreign_items::EvalContextExt as ForeignItemsEvalContextEx
 pub use crate::shims::intrinsics::EvalContextExt as IntrinsicsEvalContextExt;
 pub use crate::shims::tls::{EvalContextExt as TlsEvalContextExt, TlsData};
 pub use crate::shims::dlsym::{Dlsym, EvalContextExt as DlsymEvalContextExt};
+pub use crate::shims::env::{EnvVars as ShimsEnvVars};
```

Comment:
We don't usually have a `Shims` prefix. Please follow the naming of everything else. (That would be just `pub use crate::shims::env::EnvVars;`.)